### ISSUE TITLE
Support for RESTful API that use only Token authentication 

### ISSUE THREAD

**Issue by acandelaria1** [revsys/django-friendship#67]
I am getting errors when I attempt to Post to a Friendship endpoint. I'm getting an error for the Cross site request forgery token not being set correctly? However, I am only using token authentication for my API.

**Comment by frankwiles** [comment]
Yeah there aren't API endpoints for this project.  Would be happy for someone to contribute them, but we won't likely get around to it for quite some time ourselves.

**Comment by acandelaria1** [comment]
Alright cool. I will see if I can make time to contribute!
